pkg/webhook/validation/dynakube/namespace_selector.go

```diff
@@ -14,6 +15,7 @@ Make sure the namespaceSelector doesn't conflict with other Dynakubes namespaceS
 	errorConflictingNamespaceSelectorNoSelector = `The DynaKube does not specificy namespaces where it should inject into while another Dynakube already injects into namespaces, which is not supported.
 Make sure you have a namespaceSelector doesn't conflict with other Dynakubes namespaceSelector
 `
+	errorNamespaceSelectorMatchLabelsViolateLabelSpec = "The DynaKube's namespaceSelector contains matchLabels that are not conform to spec. A valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character."
```

Comment:
doesn't the validation give back some output that we could just use?
Something like:
`The DynaKube's namespaceSelector contains matchLabels that are not conform to spec: <What the validation gives back>`

Reply:
I actually thought about that too but wanted not to deviate from the pattern I saw, but will happily do so :)